src/PAModel/Utility/FilePath.cs

```diff
@@ -25,19 +25,21 @@ public FilePath(params string[] segments)
 
         public string ToMsAppPath()
         {
-            return string.Join("\\", _pathSegments);
+            var path = string.Join("\\", _pathSegments);
+
+            // Some paths mistakenly start with DirectorySepChar in the msapp,
```

Comment:
>mistakenly [](http://example.com/codeflow?start=26&length=10)

is this a server bug in generating the app?

Reply:
possibly I have copied over this logic from DirectoryHelper.cs, where we remove this '_' when we load app from sources. @lesaltzm do you know if this was a server bug?